modules/doc/content/newsletter/2025/2025_01.md

```diff
@@ -14,16 +9,97 @@ convergence and divergence criteria. The parsed expression may include other
 [Convergence](Convergence/index.md) objects, [Functions](Functions/index.md),
 [Post-processors](Postprocessors/index.md), and constant values.
 
+### General meshing
+
+The [CutMeshByLevelSetGenerator.md] mesh generator was added to be able to modify meshes using general functions,
+while keeping the cut mesh with a smooth first order surface. Previously, only plane cuts were supported.
+
+### Restart capabilities
+
+The [SolutionUserObject.md] can now use a fixed transformation on the time that it reads the solution at. This can be
+used to load a new solution on every time step, but at a different rate than the simulation time.
+
+The old (one timestep prior) and older (two timesteps prior) states of a variable can now be initialized in the
+[initial conditions `[ICs]` block](syntax/ICs/index.md).
+
 ## MOOSE Modules Changes
 
+### Thermal hydraulics
+
+A new closure was added that leverages functors to compute the friction coefficients in flow components,
+the wall heat transfer coefficient, and other thermal hydraulics closures.
```

Comment:
@GiudGiud what closure was this? Documentation?

Reply:
Functor closures

https://mooseframework.inl.gov/docs/source/closures/FunctorClosures.html